nucliadb/tests/search/unit/test_find_post_index.py

```diff
@@ -173,7 +188,7 @@ def expected_find_response():
                                 "id": "rid-1/f/field-a/10-20",
                                 "is_a_table": False,
                                 "labels": ["/a/title"],
-                                "order": 1,
+                                "order": 2,
```

Comment:
are we expecting different order because there was a bug?

Reply:
Yes, the multiple-match bug. If you look at the previous response, there was a missing order in the list xD